spec/controllers/activity_forms_controller_spec.rb

```diff
@@ -27,6 +27,31 @@
         end
       end
 
+      context "purpose step" do
+        render_views
+
+        before { get_step :purpose }
+
+        it "includes all notes about the title and description fields" do
+          expect(response.body).to include(t("form.hint.activity.title", level: t("page_content.activity.level.#{activity.level}")))
+          expect(response.body).to include(t("form.hint.activity.title_sid_note"))
+          expect(response.body).to include(t("form.hint.activity.description"))
+          expect(response.body).to include(t("form.hint.activity.description_sid_note"))
+        end
+
+        context "when it's a non-ODA activity" do
+          let(:activity) { create(:programme_activity, :ispf_funded, is_oda: false) }
+
+          it "includes character limit notes about the title and description fields, but not SID information" do
+            expect(response.body).to include(t("form.hint.activity.title", level: t("page_content.activity.level.#{activity.level}")))
```

Comment:
Personal rant, but I find using the translations in the specs obfuscates what's actually going on (though I understand it keeps things DRY and the rest of the project is like this!)

Reply:
Yeah, I did wonder about this - I know Cristina prefers just writing it out, and I think I do too. I kind of copied the pattern from `spec/controllers/level_b/activities/uploads_controller_spec.rb`, but since this spec doesn't have any other translations, I think let's go with explicitly writing it out to continue setting that precedent